### ISSUE TITLE
Black color when there is no data when graph is first loaded

### ISSUE THREAD

**Issue by unluckychild** [kurkle/chartjs-plugin-autocolors#32]
I update graph via ajax, but when the page/graph is first loaded and without data, it shows nothing. Its OK, but when i update graph with new set of data, there is only black color for all graphs.

**Comment by AlexanderZhirov** [comment]
Confirm. Does not work when `updating` from an empty dataset. I have a black schedule.

```
type: 'pie',
        labels: [],
        data: {
            datasets: [{
                data: [],
                label: 'Voices: ',
                borderWidth: 2
            }]
        },
        options: {
            plugins: {
                autocolors: {
                    enabled: true,
                    mode: 'data'
                }
            }
        }
```

![изображение](https://github.com/kurkle/chartjs-plugin-autocolors/assets/37277056/40c4d054-46d5-42d8-931e-68da5254b549)

**Comment by marcossegovia** [comment]
Hi team, I am facing the same issue. Is there any fix/workaround for the above issue?

**Comment by unluckychild** [comment]
Yeah, i fixed this by not using this plugin. I now use only default chartjs colors, this fixed the problem

**Comment by kurkle** [comment]
Fixed in v0.3.1
